packages/core/src/types/config/index.ts

```diff
@@ -80,6 +80,7 @@ export interface LibConfig extends RsbuildConfig {
   footer?: BannerAndFooter;
   shims?: Shims;
   dts?: Dts;
+  umdName?: string;
```

Comment:
add to https://github.com/web-infra-dev/rslib/blob/main/packages/core/src/config.ts#L1100

Reply:
I changed the `omit` function, so it's required to add new keys here in the future.